packages/react-composites/tests/browser/call/app/QueryArgs.ts

```diff
@@ -31,6 +33,7 @@ export function parseQueryArgs(): QueryArgs {
     userId: params.userId ?? '',
     groupId: params.groupId ?? '',
     token: params.token ?? '',
-    displayName: params.displayName ?? ''
+    displayName: params.displayName ?? '',
+    role: params.role as Role
```

Comment:
If they don't pass in a role should we assume they are a presenter?

Reply:
I would say no. We don't want to pass the role Presenter to the composite for non-rooms calls.

Reply:
It is an optional prop, so I don't think we need to set default value